v2/src/components/appInfoForm/index.tsx

```diff
@@ -33,14 +33,61 @@ type State = {
     netlifyApiRouteUsed: boolean,
```

Comment:
The first line of the form is not true anymore, we should remove it. In this case the styling from the first line should apple to the second line

<img width="736" alt="Screenshot 2022-08-31 at 11 33 35 AM" src="https://user-images.githubusercontent.com/18233774/187605530-c4266a89-f0c6-4dbf-92cd-27fbcec34708.png">

Reply:
updated